### PR TITLE
dtdevdoc-8406-hosted-card-fields-update

### PR DESCRIPTION
**Pull request by devapplepaypal**
Jira ticket: https://paypal.atlassian.net/browse/DTDEVDOC-8406

* Promoted `/advanced-integration/beta` to `/advanced-integration/v2`.
* Cleaned up the `/advanced-integration` folder.
* Updated the `README.md` for both `/advanced-integration/v1` and `/advanced-integration/v2`.

### CODE REVIEW

File: advanced-integration/README.md [hunk 1]
```diff
@@ -1,9 +1,14 @@
-# Advanced Integration Example
+# Advanced Checkout Integration Example
 
-This folder contains example code for an Advanced PayPal integration using both the JS SDK and Node.js to complete transactions with the PayPal REST API.
+This folder contains example code for an advanced Checkout PayPal integration using both the JavaScript SDK and Node.js to complete transactions with the PayPal REST API.
+
+* `v2` contains sample code for the current advanced Checkout integration. This includes guidance on using Hosted Card Fields.
+* `v1` contains sample code for the legacy advanced Checkout integration. Use `v2` for new integrations.
```
Comment (gregjopa) [inline advanced-integration/README.md:6]:
```suggestion
* [v1](v1/README.md) contains sample code for the legacy advanced Checkout integration. Use `v2` for new integrations.
```

**Comment by MALIKSHAHID786** [comment]
> Jira ticket: https://paypal.atlassian.net/browse/DTDEVDOC-8406
> 
> * Promoted `/advanced-integration/beta` to `/advanced-integration/v2`.
> * Cleaned up the `/advanced-integration` folder.
> * Updated the `README.md` for both `/advanced-integration/v1` and `/advanced-integration/v2`.

File: advanced-integration/README.md [hunk 2]
```diff
@@ -1,9 +1,14 @@
-# Advanced Integration Example
+# Advanced Checkout Integration Example
 
-This folder contains example code for an Advanced PayPal integration using both the JS SDK and Node.js to complete transactions with the PayPal REST API.
+This folder contains example code for an advanced Checkout PayPal integration using both the JavaScript SDK and Node.js to complete transactions with the PayPal REST API.
```
Comment (e-tu) [inline advanced-integration/README.md:3]:
```suggestion
This folder contains example code for a PayPal advanced Checkout integration using both the JavaScript SDK and Node.js to complete transactions with the PayPal REST API.
```

File: advanced-integration/v1/README.md [hunk 3]
```diff
@@ -1,11 +1,14 @@
 # Advanced Integration Example
 
-This folder contains example code for an Advanced PayPal integration using both the JS SDK and Node.js to complete transactions with the PayPal REST API.
+This folder contains example code for [version 1](https://developer.paypal.com/docs/checkout/advanced/integrate/sdk/v1) of an advanced Checkout PayPal integration using the JavaScript SDK and Node.js to complete transactions with the PayPal REST API.
```
Comment (e-tu) [inline advanced-integration/v1/README.md:3]:
```suggestion
This folder contains example code for [version 1](https://developer.paypal.com/docs/checkout/advanced/integrate/sdk/v1) of a PayPal advanced Checkout integration using the JavaScript SDK and Node.js to complete transactions with the PayPal REST API.
```

File: advanced-integration/v2/README.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Advanced Integration Example
+
+This folder contains example code for [version 2](https://developer.paypal.com/docs/checkout/advanced/integrate/) of an advanced Checkout PayPal integration using the JavaScript SDK and Node.js to complete transactions with the PayPal REST API.
```
Comment (e-tu) [inline advanced-integration/v2/README.md:3]:
```suggestion
This folder contains example code for [version 2](https://developer.paypal.com/docs/checkout/advanced/integrate/) of a PayPal advanced Checkout integration using the JavaScript SDK and Node.js to complete transactions with the PayPal REST API.
```

File: advanced-integration/v2/README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+Version 2 is the current advanced Checkout integration, and includes Hosted Card Fields.
```
Comment (e-tu) [inline advanced-integration/v2/README.md:5]:
```suggestion
Version 2 is the current advanced Checkout integration, and includes hosted card fields.
```

File: advanced-integration/v2/README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+2. Rename `.env.example` to `.env` and update `PAYPAL_CLIENT_ID` and `PAYPAL_CLIENT_SECRET`.
+3. Replace `test` in [client/checkout.html](client/checkout.html) with your app's client-id
+4. Run `npm install`
+5. Run `npm start`
+6. Open http://localhost:8888
+7. Enter the credit card number provided from one of your [sandbox accounts](https://developer.paypal.com/dashboard/accounts) or [generate a new credit card](https://developer.paypal.com/dashboard/creditCardGenerator)
+
+## Examples
+
+The documentation for advanced Checkout integration using JavaScript SDK includes additional sample code in the following sections:
```
Comment (e-tu) [inline advanced-integration/v2/README.md:19]:
```suggestion
The documentation for an advanced Checkout integration using the JavaScript SDK includes additional sample code in the following sections:
```

**Review by e-tu — CHANGES_REQUESTED** [review]

**Review by gregjopa — APPROVED** [review]

**Comment by shango420** [comment]
> https://paypal.me/xhangtech?country.x=MX&locale.x=es_XC

**Comment by shango420** [comment]
> > https://paypal.me/xhangtech?country.x=MX&locale.x=es_XC

c4ee30fb101f3b2ea4a3665a37c7beca65fb02b5

**Pull request merged**